src/ess/spectroscopy/indirect/__init__.py

```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+
```

Comment:
Just curious why do we add these headers? We don't *need* to add this given there is a root level license file.

Reply:
To make it easier for people to see the license if they get to a file with a direct link or look at it on disk. And in case a file gets copied out of a repo and sent to someone.